pkg/plugins/plugindef/plugindef.cue

```diff
@@ -120,6 +120,24 @@ seqs: [
 					...
 				}
 
+				#ExtensionsLink: {
+					// Unique identifier of the plugin link extension
+					id:          string
+					description: string
+					// Relative path to the plugin
```

Comment:
Relative path to what plugin? i can't figure out from these comments what's actually supposed to be declared here

Reply:
The plugin that you are adding your extension points to.